deepchecks/utils/abstracts/confusion_matrix_abstract.py

```diff
@@ -9,40 +9,46 @@
 # ----------------------------------------------------------------------------
 #
 """The confusion_matrix_report check module."""
+import textwrap
 from typing import List
 
 import numpy as np
 import pandas as pd
 import plotly.graph_objects as go
+from plotly.subplots import make_subplots
 from sklearn.metrics import confusion_matrix
 
 from deepchecks import ConditionCategory, ConditionResult
 from deepchecks.core import CheckResult
 from deepchecks.core.errors import DeepchecksValueError
-from deepchecks.utils.strings import format_number_if_not_nan, format_percent
+from deepchecks.utils.strings import format_percent
 
 __all__ = ['create_confusion_matrix_figure', 'run_confusion_matrix_check']
 
+MAX_WORST_CLASSES_TO_DISPLAY = 3
+MAX_TOP_CLASSES_TO_DISPLAY = 3
```

Comment:
Redundant? at least i couldn't find where it is used

Reply:
Done